### PR TITLE
benchmark: simplify implementation, fix comments

### PR DESCRIPTION
**Pull request by seliopou**
The pure benchmark code was defining its own skip_many combinator, which is already defined the main module. A comment in that benchmark code also contained a typo, which was corrected.

### CODE REVIEW

File: benchmarks/pure_benchmark.ml [hunk 1]
```diff
@@ -13,16 +13,11 @@ let read file =
...
-  let open Angstrom in
-  fix (fun m ->
-    (p *> m) <|> return ())
-
 (* For input files involving trailing numbers, .e.g, [http-requests.txt.100],
  * go into the [benchmarks/data] directory and use the [replicate] script to
  * generate the file, i.e.,
  *
- *   [./replicate http-requests.txt.100]
+ *   [./replicate http-requests.txt 100]
```
Comment (rgrinberg) [inline benchmarks/pure_benchmark.ml]: Is this change intended? The ocamldoc right above still contains the period

Comment (seliopou) [inline benchmarks/pure_benchmark.ml]: The change was intended. First argument is file name, second is how many times it should appear in the output. The output file name is the concatenation of the two arguments, with a period in between.
